src/components/transactions/FlowCommons/TxModalDetails.tsx

```diff
@@ -137,7 +137,7 @@ export const DetailsNumberLineWithSub = ({
           </>
         ) : (
           <>
-            <Box sx={{ display: 'flex', alignItems: 'center' }}>
+            <Box sx={{ display: { xs: 'block', lg: 'flex' }, alignItems: 'center' }}>
```

Comment:
With this change, small and medium screens are going to also be set to `block` since that is the default value.

I'm wondering if there's a different approach we could take here though. When the amount to repay is less than 1, it defaults to showing 7 decimal places. Maybe on xs screen sizes we could just set the number of decimals to 2 so it won't overflow? Maybe a question for @iamanastasia